ui/public/src/main.js

```diff
@@ -53,6 +55,13 @@ export default async function main() {
       }
       case 'walletUpdatePurses': {
         // TODO: handle appropriately
+        const purses = JSON.parse(obj.data);
+        const tokenPurse = purses.find(
+          ({ brandBoardId }) => brandBoardId === TOKEN_BRAND_BOARD_ID,
+        );
+        if (tokenPurse && tokenPurse.pursePetname) {
+          pursePetname = tokenPurse.pursePetname;
+        }
```

Comment:
Looks good! Could we add some comments to explain to a new user what is happening here?